### ISSUE TITLE
Resource with Identifier: 6bcaedc5 not found.

### ISSUE THREAD

**Issue by skeusgen** [lexevs/lexevs#22]
I installed LexEVS 6.3 (local runtime, lexevscts2 and uriresolver) on a tomcat (apache-tomcat-7.0.63, jdk1.7.0_60). 

Then I imported the automobiles sample codesystem and the corresponding value sets via the admin guis. In the value set admin gui I can see the resolved value sets (e. g. GM). Then I restarted my tomcat.

But when I request the valueset via rest service call (/lexevscts2/valueset/GM/resolution), I get the error 'Resource with Identifier: 6bcaedc5 not found.'. 6bcaedc5 is the definition of the value set. Why can`t I resolve value sets? Is there something missing in my installation?

**Comment by hsbauer** [comment]
This can be a challenging tech stack to install.  Let's see if I can understand where you are at.  Do you have any calls to the rest service working at this point?  i.e. can you resolve a call to codesystemversions on your base url?

**Comment by skeusgen** [comment]
Yes I can:
/lexevscts2/codesystemversions works fine. I get my two codesystems imported before.
/lexevscts2/codesystem/Automobiles/version/1.0 works fine
/lexevscts2/codesystem/Automobiles/version/1.0/entities shows all entities
/lexevscts2/valueset/GM shows the definitionid
and /lexevscts2/valueset/GM/definition/6bcaedc5 the definition itself.

But /lexevscts2/valueset/GM/definition/6bcaedc5/resolution results in the error described.

**Comment by hsbauer** [comment]
Resolved value sets are a special subset of the value set model element and must be loaded separately.  So if you aren’t seeing a response to resolvedvaluesets, then you may not have resolved your value set definitions into the static resolved value sets in lexevs.  These are loaded in the regular lbGUI admin application.  Have you taken this step yet?

**Comment by skeusgen** [comment]
No, we only loaded Valuesets using the ValueSet Console. But how can I get all values defined in a value set as the GM sample?

**Comment by hsbauer** [comment]
My mistake.  I see you are attempting a dynamic resolution of this valueset.  Can you send along a full stack trace of your error from the tomcat logs?

**Comment by hsbauer** [comment]
We've recreated your error here.  I'd suggest you load the value set definition as a resolved value set.  I'm not sure if this is working as designed in CTS2, but I suspect it may be.

**Comment by skeusgen** [comment]
OK, how can I do that? I mean, I thought in the lbGUI I can only load codesystems. How can I load value sets here? It would be very helpful if you can provide a sample. I am not very familiar with the different import formats...

**Comment by skeusgen** [comment]
ok, I exported the GM value set as resolved lex grid file from the value set console and imported the file into the lbGUI. Now I get the resolved value set entries with this request:
/lexevscts2/codesystem/gm/version/1.0/entities 

But I would like to request it as value set not as codesystem. Thanks for help!

**Comment by skeusgen** [comment]
How is it possible to load a value set and request it with its values using CTS2? I thought resolve has to do this?

**Comment by hsbauer** [comment]
We have created an issue to track this request for a future release.  JIRA item LEXEVS-1156.
https://tracker.nci.nih.gov/browse/LEXEVS-1156
